packages/services/storage/src/index.ts

```diff
@@ -4054,6 +4058,28 @@ const schemaCheckSQLFields = sql`
   , "manual_approval_user_id" as "manualApprovalUserId"
 `;
 
+const schemaCheckSQLFieldsFallback = sql`
+  "sc"."id"
+  , to_json("sc"."created_at") as "createdAt"
+  , to_json("sc"."updated_at") as "updatedAt"
+  , coalesce("sc"."schema_sdl", "sv"."base_schema") as "schemaSDL"
```

Comment:
I am not sure if this is a proper fallback TBH

Reply:
The fallback would be `sv.composite_schema_sdl`

Reply:
But it needs to be done in the business logic because a schema check has no SDL if the composition failed.

Reply:
added to zod object.transform  on SuccessfulSchemaCheckPartialModel